Modules/_csv.c

```diff
@@ -923,6 +917,10 @@ Reader_clear(ReaderObj *self)
     Py_CLEAR(self->dialect);
     Py_CLEAR(self->input_iter);
     Py_CLEAR(self->fields);
+    if (self->field != NULL) {
```

Comment:
Same as https://github.com/python/cpython/pull/26074/files#r631232338

Reply:
Only `tp_dealloc` should be freeing memory.

Reply:
Fixed in fe74fc2dd9ca1a59e3f12a0bb5adaf0c36834b85